docs-markdown/src/test/suite/controllers/preview-controller.test.ts

```diff
@@ -0,0 +1,58 @@
+import * as chai from 'chai';
+import { resolve } from 'path';
+import { commands, window } from 'vscode';
+import { previewTopic, seoPreview } from '../../../controllers/preview-controller';
+import * as common from '../../../helper/common';
+import * as telemetry from '../../../helper/telemetry';
+import { loadDocumentAndGetItReady, sleep } from '../../test.common/common';
+import * as vscode from 'vscode';
+
+// tslint:disable-next-line: no-var-requires
+import sinon = require('sinon');
+
+const expect = chai.expect;
+
+suite('Preview Controller', () => {
+	teardown(() => {
+		chai.spy.restore(common);
+	});
+	suiteTeardown(async () => {
+		await commands.executeCommand('workbench.action.closeAllEditors');
+	});
+	test('previewTopicCommands', () => {
+		const controllerCommands = [
+			{ command: previewTopic.name, callback: previewTopic },
+			{ command: seoPreview.name, callback: seoPreview }
+		];
+	});
+	test('noActiveEditorMessage', () => {
+		const spy = chai.spy.on(common, 'noActiveEditorMessage');
+		previewTopic();
+		expect(spy).to.have.been.called();
+	});
+	test('isMarkdownFileCheck', async () => {
+		const filePath = resolve(__dirname, '../../../../../src/test/data/repo/articles/preview.md');
+		await loadDocumentAndGetItReady(filePath);
+		const stub = sinon.stub(telemetry, 'sendTelemetryData');
```

Comment:
can you put this line in a suiteSetup similar to this test https://github.com/microsoft/vscode-docs-authoring/blob/d9f16d58c84427179d24efde462ffac9658a03f4/docs-markdown/src/test/suite/controllers/image-controller.test.ts#L40. You will also need the suiteTearDown https://github.com/microsoft/vscode-docs-authoring/blob/d9f16d58c84427179d24efde462ffac9658a03f4/docs-markdown/src/test/suite/controllers/image-controller.test.ts#L35